mailbox/api/src/test/java/org/apache/james/mailbox/MailboxManagerTest.java

```diff
@@ -566,6 +572,237 @@ public void searchShouldAllowNamespaceFiltering() throws MailboxException {
     }
 
     @Test
+    @Ignore
+    public void searchForMessageShouldReturnMessagesFromAllMyMailboxesIfNoMailboxesArePrecised() throws MailboxException {
+        boolean isRecent = false;
+
+        session = mailboxManager.createSystemSession(USER_1);
+
+        MailboxPath folderForCacahuete = MailboxPath.forUser(USER_1, "CACAHUETE");
```

Comment:
cacahueteFolder?

Reply:
Random name do you have an other proposition, the idea here is just to have a folder A and a folder B and for each of them to have corresponding variable for mail, path, etc..., etc...

Reply:
my suggestion is to have this naming in the right order: cacahueteFolder instead of folderForCacahuete